action/protocol/account/transfer.go

```diff
@@ -39,7 +39,7 @@ func (p *Protocol) handleTransfer(ctx context.Context, tsf *action.Transfer, sm
 		return nil, errors.Wrapf(err, "failed to load or create the account of sender %s", actionCtx.Caller.String())
 	}
 
-	gasFee := big.NewInt(0).Mul(tsf.GasPrice(), big.NewInt(0).SetUint64(actionCtx.IntrinsicGas))
+	gasFee, baseFee := protocol.SplitGas(ctx, tsf.AbstractAction)
```

Comment:
splition should be same for all types of actions?

Reply:
this will be in next PR